packages/dashboard-core-plugins/src/panels/MarkdownPanel.tsx

```diff
@@ -123,10 +128,6 @@ export class MarkdownPanel extends Component<
     }
   }
 
-  getClosedMarkdowns = memoize((closedPanels: ClosedPanels) =>
-    closedPanels.filter(panel => panel.component === 'MarkdownPanel').reverse()
-  );
-
```

Comment:
We still want to memoize this, but it can call into the `MarkdownUtils.getClosedMarkdowns` method